src/Controls/samples/Directory.Build.targets

```diff
@@ -1,4 +1,4 @@
 <Project>
-  <Import Project="Maui.InTree.targets" Condition=" '$(UseMaui)' != 'true' " />
+  <Import Project="../../Maui.InTree.targets" Condition=" '$(UseMaui)' != 'true' " />
```

Comment:
Moved the InTree up a level so we could re-use them in other folders instead of copying.